apps/web/src/pages/HumanResources/PlatformResourcesForProfessionalsPage.tsx

```diff
@@ -0,0 +1,475 @@
+import BriefcaseIcon from "@heroicons/react/24/outline/BriefcaseIcon";
+import WrenchScrewdriverIcon from "@heroicons/react/24/outline/WrenchScrewdriverIcon";
+import PresentationChartBarIcon from "@heroicons/react/24/outline/PresentationChartBarIcon";
+import CloudIcon from "@heroicons/react/24/outline/CloudIcon";
+import QuestionMarkCircleIcon from "@heroicons/react/24/outline/QuestionMarkCircleIcon";
+import { useIntl } from "react-intl";
+import { ReactNode, useState } from "react";
+
+import {
+  Accordion,
+  Button,
+  CardFlat,
+  Container,
+  Heading,
+  Link,
+  Ul,
+} from "@gc-digital-talent/ui";
+import {
+  getLocale,
+  navigationMessages,
+  uiMessages,
+} from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/Hero";
+import SEO from "~/components/SEO/SEO";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+import pageTitles from "~/messages/pageTitles";
+
+const externalLink = (chunks: ReactNode, href: string) => (
+  <Link newTab external href={href}>
+    {chunks}
+  </Link>
+);
+
+const internalLink = (chunks: ReactNode, href: string) => (
+  <Link href={href}>{chunks}</Link>
+);
+
+const FAQ_ID = {
+  PSC_GCDT: "psc-gcdt-aware",
+  SKILLS_BASED: "skills-based-recruitment",
+  TEMPLATES: "job-advertisement-templates",
+  PREQUAILIFIED_TALENT: "prequalified-talent",
+  RECRUITMENT_PROCESS: "recruitment-process",
+  COMMUNITIES: "functional-communities",
+} as const;
+
+const faqIds = Object.values(FAQ_ID);
+
+export const Component = () => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [openFaq, setOpenFaq] = useState<string[]>([]);
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Platform resources for HR professionals",
+    id: "yIoVwH",
+    description: "Title for professional HR resources page",
+  });
+
+  const desc = intl.formatMessage({
+    defaultMessage:
+      "Discover the tools available to support HR and recruitment experts across the Government of Canada.",
+    id: "bcyIgm",
+    description: "Description of profressional HR resources page",
+  });
+
+  const crumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        url: paths.professionalHRResources(),
+        label: pageTitle,
+      },
+    ],
+  });
+
+  const toggleFaq = () => {
+    if (openFaq.length > 0) {
+      setOpenFaq([]);
+    } else {
+      setOpenFaq(faqIds);
+    }
+  };
+
+  return (
+    <>
+      <SEO title={pageTitle} description={desc} />
+      <Hero title={pageTitle} subtitle={desc} crumbs={crumbs} />
+      <Container>
+        <Heading icon={BriefcaseIcon} color="primary" className="font-normal">
+          {intl.formatMessage({
```

Comment:
Per the spacing discussion in the other PR. As an aside, I think I know what the margin value should be for the heading according to Figma specs.

Given our Tailwind migration was value = x * 6 [ref example](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14019/files#diff-69f70ec5e3201968e29d7f80acf17e57b62e41f71982cd36a2f3e32bc7ff0511)
And the below Figma specs

<img width="676" height="575" alt="Image" src="https://github.com/user-attachments/assets/66dab3df-a4af-4316-9098-2e74d5d4abde" />

Heading component should probably be `mt-18` rather than `mt-12`
Of course, we have to get around to component updating first...

Reply:
Yeah, I really think we sohuld be adressing the base component here :melting_face: 

If we have to, I can adress this but its not great that we keep sprinkling overrides everywhere with our headings. It would be really nice to just use the base heading component and have it work with little to no overrides.

Reply:
Would be nice for the heading to have the exact spacing needed, yeah